rules/windows/privilege_escalation_printspooler_suspicious_spl_file.toml

```diff
@@ -135,7 +136,8 @@ file where host.os.type == "windows" and event.type != "deletion" and
              "?:\\Program Files\\*.exe",
              "?:\\Program Files (x86)\\*.exe",
              "?:\\PROGRA~1\\*.exe",
-             "?:\\PROGRA~2\\*.exe")
+             "?:\\PROGRA~2\\*.exe",
+             "?:\\Windows\\System32\\rundll32.exe")
```

Comment:
A bit broad, but I saw many hosts executing `printui.dll,PrintUIEntryDPIAware`, which seems to be the cause here.